imageroot/systemd/user/freepbx.service

```diff
@@ -64,6 +69,9 @@ ExecStart=/usr/bin/podman run \
     --hostname=${NETHVOICE_HOST} \
     ${NETHVOICE_FREEPBX_IMAGE}
 
+# Wait till asterisk is active
+ExecStartPost=podman exec asterisk bash -c "while [[ $$(/usr/sbin/asterisk -rx 'core show version' 2>/dev/null) != Asterisk* ]]; do ((++attempt<30)) || exit 2; sleep 1 ; done"
```

Comment:
Why do you need to wait?

Reply:
to make sure asterisk is full started before running dependent services